tests/testthat/test-snapshots.R

```diff
@@ -0,0 +1,34 @@
+testthat::local_edition(3)
+quarto_docs <- list.files(
+    "../../book/",
```

Comment:
Using `..` in paths is quite dangerous in testing files because this has different path in R CMD CHECK, testthat::test() and probably more scenarios. `testhat` should have a solution for this